src/Core/OAuth/OAuth2/OAuth2LoginHelper.php

```diff
@@ -82,6 +85,12 @@ class OAuth2LoginHelper
      */
     private $faultHandler = false;
 
+    /**
+     * Get the Logging component for the REST service
+     * @var LogRequestsToDisk
+     */
+    protected $RequestLogging;
```

Comment:
Can you please camelCase it across the file?

Reply:
Agree, just copied what you have in another module

Reply:
$RequestLogging changed to camelCase